src/Microsoft.AspNet.Razor/Parser/CSharpCodeParser.Statements.cs

```diff
@@ -13,6 +13,9 @@ namespace Microsoft.AspNet.Razor.Parser
 {
     public partial class CSharpCodeParser
     {
+        private static readonly Func<CSharpSymbol, bool> IsValidStatementSpacingSymbol =
+            IsSpacingToken(includeNewLines: true, includeComments: true);
```

Comment:
That line of code would be identical either way. I'm talking about having to `F12` twice to reach the real code.